muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/matriculation-eligibility-row/matriculation-eligibility-row.tsx

```diff
@@ -114,20 +66,21 @@ class MatriculationEligibilityRow extends React.Component<MatriculationEligibili
    * 
    * @returns matriculation subject name or empty string if not found 
    */
+  
   getName(): string {
     return this.props.i18n.text.get(`plugin.records.hops.matriculationSubject.${this.props.code}`);
   }
-
+  
   /**
    * Returns list text for student matriculation eligibility
    * 
    * @returns list text for student matriculation eligibility
    */
   getEligibleText(): string {
-    switch (this.state.eligible) { 
-      case EligbleEnum.TRUE:
+    switch (this.props.subjectEligibility.egilibility) { 
+      case "TRUE":
```

Comment:
Not necessary but we could use something else than TRUE or FALSE, maybe, ELEGIBLE and NOT_ELEGIBLE, because these are strings, I know that we have a UNKNOWN case, but it makes it that it's not very clear why we didn't use the native boolean value, and makes it hard to read the code later when we didn't know what this all was about.